scripts/legislation/africa/liberia_english.sh

```diff
@@ -0,0 +1,316 @@
+    amend_error_in_article 85 'aquana' 'aquaria' | \
+    amend_error_in_article 85 'orgamsms' 'organisms' | \
+    amend_error_in_article 85 '1V.' '[iv]' | \
+    amend_error_in_article 85 '\[1\] germplasm' '[i] germplasm' | \
+    amend_error_in_article 85 '\[1\] the' '[i] the' | \
+    #Article 86
+    sed -E 's/Resou rces/Resources/' | \
+    amend_error_in_article 86 'nonresidents' 'non residents' | \
+    amend_error_in_article 86 'ofDistrict' 'of District' | \
+    #Article 87
+    sed -E 's/55 Land/Land/' | \
+    amend_error_in_article 87 'with he' 'with the' | \
+    #Article 89
+    amend_error_in_article 89 '\[i\] The' '[1] The' | \
+    #Article 90
+    amend_error_in_article 90 '\(6\)' '[6]' | \
+    #Article 91
+    amend_error_in_article 91 'COUli' 'Court' | \
+    amend_error_in_article 91 ' hall ' ' shall ' | \
+    #Article 92
+    amend_error_in_article 92 '\(l\)' '[1]' | \
+    #Article 93
+    amend_error_in_article 92 'Action by' '\n\nAction by' | \
+    sed -E 's/Article Environmental Restoration Order/Environmental Restoration Order\n(93)/' | \
+    amend_error_in_article 93 'to he' 'to the' | \
+    #Article 94
+    amend_error_in_article 94 '\(93\) and \(94\)' '93 and 94' | \
+    #Article 95
+    amend_error_in_article 95 '21 \(2\)' '21[2]' | \
+    amend_error_in_article 95 'with he' 'with the' | \
+    amend_error_in_article 95 'machi nery' 'machinery' | \
+    amend_error_in_article 95 'uncler' 'under' | \
+    amend_error_in_article 95 'an the' 'and the' | \
+    amend_error_in_article 95 'or \[1\]' 'or [l]' | \
+    #Article 99
+    sed -E ':start;s/^(\(99\).*)intemational/\1international/;t start' | \
+    amend_error_in_article 99 'conceming' 'concerning' | \
+    amend_error_in_article 99 'than he ' 'than the ' | \
+    amend_error_in_article 99 'and Whether' 'and whether' | \
+    #Article 105
+    amend_error_in_article 105 'briefin' 'brief in' | \
+    amend_error_in_article 105 'an is' 'and is' | \
+    #ARticle 108
+    amend_error_in_article 108 'the act a' 'the act is' 
+}
+
+function amend_errors_in_headers {
+  sed -E 's/PART I /\n\nPART I /' | \
+    sed -E 's/^(PART .*)\[•\]/\1-/g' | \
+    sed -E 's/15 PART II \[•\]/PART II -/' | \
+    sed -E 's/PART V \[•\]/\n\nPART V -/' | \
+    sed -E 's/PART IX-/PART IX - /' | \
+    sed -E 's/([A-Z]{2} )([A-Z][a-z])/\1\n\n\2/g' | \
+    sed -z "s/\n\nLiberia's natural/Liberia's natural/" | \
+    sed -E 's/THE MANAGEMEN /THE MANAGEMENT /' 
+}
+
+function preprocess_state_and_language_input_file {
+  if [ "$#" -ne 2 ] ; then
+    echo_usage_error "$*" '<input_file_path> <language>'
+    return 1
+  fi
+  local input_file_path="$1"
+  local language="$2"
+
+  cat "$input_file_path" | \
+    remove_all_text_before_first_header | \
+    format_article_numbers_and_titles | \
+    apply_common_transformations_to_stdin "$language" | \
+    move_article_titles_above_article_bodies | \
+    remove_all_text_after_last_article | \
```

Comment:
Please move the `remove_all_text_after_last_article` function body below the `move_article_titles_above_article_bodies` function body.